packages/ui-kit/package.json

```diff
@@ -43,6 +43,7 @@
     "@babel/preset-env": "~7.21.4",
     "@rocket.chat/eslint-config-alt": "workspace:~",
     "@rocket.chat/prettier-config": "workspace:~",
+    "@rocket.chat/icons": "workspace:~",
```

Comment:
If the types from this package end in the final `dist/**/*.d.ts` files, then it should have been listed as peer dependency too.